provide-storage/backup-storage-provider/run-a-bsp-node.md

```diff
@@ -0,0 +1,384 @@
+| **RAM** | 32 GB DDR4 ECC |
+| **Storage (System)** | 500 GB NVMe SSD (chain data) |
+| **Storage (User Data)** | 1 TB NVMe SSD or HDD (minimum) |
+| **Network** | 500 Mbit/s symmetric |
+
+#### Important Considerations
+
+- **Separate storage volumes**: Keep chain data and user data on separate volumes for better I/O performance
+- **Storage expansion**: Plan for growth; user data storage should be easily expandable
+- **max-storage-capacity**: Set this CLI flag to **80% of available physical disk space** to leave headroom for filesystem overhead and temporary files
+- **Cloud compatible**: BSPs can run effectively on cloud VPS with dedicated storage volumes
+- **Proof submission**: Ensure reliable network connectivity for timely proof submissions
+
+### Deposit Requirements
+
+- **Base Deposit**: 100 MOCK (`SpMinDeposit`)
+- **Per GiB**: 2 MOCK (`DepositPerData`)
+- **Formula**: 100 + (`capacity_in_gib` × 2) + buffer
+
+Examples:
+
+- **800 GiB capacity**: 100 + (800 × 2) = 1,700 MOCK required (1,800 MOCK recommended)
+- **1.6 TiB capacity**: 100 + (1,638 × 2) = 3,376 MOCK required (3,500+ MOCK recommended)
+
+The deposit is held (reserved) from your account when you start the BSP registration process and remains held while you operate as a BSP. The deposit is returned when you deregister as a BSP.
+
+!!! note
+    Your BSP account must be funded before BSP registration
+
+## Project Structure
+
+This is how the project structure will look like, once everything is set up.
+
+```
+datahaven-bsp-node/
+├── datahaven-node
+├── bsp-data
+│   └── chains
+│   │   ├── datahaven_stagenet_local
+│   │   │   └── keystore
+│   │   └── datahaven_testnet
+│   │       └── keystore
+│   bsp-storage    
+│   └── storagehub    
+│       ├── file_storage
+│       └── forest_storage      
+├── Dockerfile
+├── docker-compose.yml
+└── bsp.log    
+```
+
+## Project Setup
+
+Before running a BSP node, you will need to obtain the `datahaven-node` client binary and the chain specifications for the network you want to join.
+
+### Download Latest Client Release
+
+From the [Releases](https://github.com/datahaven-xyz/datahaven/releases){target=\_blank} section of the DataHaven repo, the latest version of the [`datahaven-node` binary](https://github.com/datahaven-xyz/datahaven/releases/download/{{ networks.testnet.client_version }}/datahaven-node){target=_\blank} can be found. Currently, the latest version is {{ networks.testnet.client_version }}.
+
+Make sure to download it in the root of your `datahaven-bsp-node` folder.
+
+### Download Desired Chain Specs
+
+Next, [download the testnet chain specs](/downloads/datahaven-testnet-raw-specs.json){target=\_blank} and include them in the root of your project. Make sure the file is called `datahaven-testnet-raw-specs.json`. The specs you use dictate to which DataHaven network your BSP will connect and interact with.
+
+## Docker Configuration for the BSP Node
```

Comment:
This should be action-oriented; something like "Configure Docker for the BSP Node"